index.html

```diff
@@ -1,39 +1,348 @@
+              dummy text.
+            </p>
+            <div>
+              <ul class="languages">
+                <li><img src="./images/css.png" alt="" /></li>
+                <li><img src="./images/html.png" alt="" /></li>
+                <li><img src="./images/bootstrap.png" alt="" /></li>
+                <li><img src="./images/ruby.png" alt="" /></li>
+              </ul>
             </div>
-            <div><img src="./Images/Group.png" alt="Background Image"></header></div>
-        </header>
-       
-       
+            <button>See Project</button>
+          </div>
+        </div>
+      </div>
+
+      <!-- projects -->
+      <div class="projects-grid">
+        <div class="project">
+          <div class="project-header">
+            <h1>Professional Art Printing Data</h1>
+          </div>
+          <div>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+          </div>
+          <div>
+            <ul class="projectLanguages">
+              <li><img src="./images/Tag (4).png" alt="" /></li>
+              <li><img src="./images/Tag (5).png" alt="" /></li>
+              <li><img src="./images/Tag (6).png" alt="" /></li>
+            </ul>
+          </div>
+          <div>
+            <button>See Project</button>
+          </div>
+        </div>
+
+        <div class="project">
+          <div class="project-header">
+            <h1>Professional Art Printing Data</h1>
+          </div>
+          <div>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+          </div>
+          <div>
+            <ul class="projectLanguages">
+              <li><img src="./images/Tag (4).png" alt="" /></li>
+              <li><img src="./images/Tag (5).png" alt="" /></li>
+              <li><img src="./images/Tag (6).png" alt="" /></li>
+            </ul>
+          </div>
+          <div>
+            <button>See Project</button>
+          </div>
+        </div>
+
+        <div class="project">
+          <div class="project-header">
+            <h1>Professional Art Printing Data</h1>
+          </div>
+          <div>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+          </div>
+          <div>
+            <ul class="projectLanguages">
+              <li><img src="./images/Tag (4).png" alt="" /></li>
+              <li><img src="./images/Tag (5).png" alt="" /></li>
+              <li><img src="./images/Tag (6).png" alt="" /></li>
+            </ul>
+          </div>
+          <div>
+            <button>See Project</button>
+          </div>
+        </div>
```

Comment:
- [ Optional ] As requested by the previous reviewer kindly implemented the requested changes , You must replicate the design as much as possible (e.g., font, colors, images, text, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Your Project section Look likes this : 

![Capture](https://user-images.githubusercontent.com/93598189/206871991-43ded63d-1005-4a92-a813-b492af6649c7.PNG)

The Figma design Look likes this : 

![Capture](https://user-images.githubusercontent.com/93598189/206872034-c1e6cf96-85c7-46a2-a95f-01a8aa64125e.PNG)